tests/pytests/unit/utils/win_dacl/test_get_name.py

```diff
@@ -0,0 +1,113 @@
+"""
+tests.pytests.unit.utils.win_dacl.test_get_name
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Test the get_name function in the win_dacl utility module
+"""
+# Python libs
+import pytest
+
+# Salt libs
+import salt.exceptions
+import salt.utils.win_dacl
+
+# Third-party libs
+try:
+    import win32security
+
+    HAS_WIN32 = True
+except ImportError:
+    HAS_WIN32 = False
+
+pytestmark = [
+    pytest.mark.windows_whitelisted,
+    pytest.mark.skip_unless_on_windows,
+]
+
+
+@pytest.mark.skipif(not HAS_WIN32, reason="Requires Win32 libraries")
+def test_get_name_normal_name():
+    """
+    Test get_name when passing a normal string name
+    """
+    result = salt.utils.win_dacl.get_name("Administrators")
+    expected = "Administrators"
+    assert result == expected
+
+
+@pytest.mark.skipif(not HAS_WIN32, reason="Requires Win32 libraries")
+def test_get_name_mixed_case():
+    """
+    Test get_name when passing an account name with mixed case characters
+    """
+    result = salt.utils.win_dacl.get_name("adMiniStrAtorS")
+    expected = "Administrators"
+    assert result == expected
+
+
+@pytest.mark.skipif(not HAS_WIN32, reason="Requires Win32 libraries")
+def test_get_name_sid():
+    """
+    Test get_name when passing a sid string
+    """
+    result = salt.utils.win_dacl.get_name("S-1-5-32-544")
+    expected = "Administrators"
+    assert result == expected
+
+
+@pytest.mark.skipif(not HAS_WIN32, reason="Requires Win32 libraries")
+def test_get_name_sid_object():
```

Comment:
You should be able to condense almost all of these tests into one test and make use of `@pytest.mark.parametrize`

Reply:
Done